### DISCUSSION TITLE
📔 Documentation

### DISCUSSION THREAD

**Discussion by yandeu** [enable3d/enable3d#117] [General]
You will find the documentation here:
https://enable3d.io/docs.html

There is clearly a lot to document yet.

If you want to help, feel free to [improve it](https://github.com/enable3d/enable3d-website/tree/master/src/docs).

The documentation should primarily descript the use of the Standalone Framework, with hints on how to adapt certain functionalities for other implementations (Phaser extension, Physics plugin for Three.js, headless mode on node.js).

Thanks for your help ❤️

**Comment by N8python** [comment]
I still don't feel that proficient with the library - after a few more weeks of working with it I will definitely start on the docs!
